src/components/adaptive-cards/Component/Component.jsx

```diff
@@ -49,9 +49,29 @@ UnknownComponent.defaultProps = {
  * @returns {object} JSX of the component
  */
 export default function Component({data}) {
+  const [cssClasses, sc] = webexComponentClasses('component', undefined);
+  const classes = [];
+  const getClass = (key, value) => sc(`${key}--${value}`);
+
+  for (const [key, value] of Object.entries(data)) {
+    switch (key) {
+      case 'separator': break;
+      case 'spacing':
+      case 'isVisible':
+        classes.push(getClass(key, value));
+        break;
+      default:
+        console.log('[Component]', 'Unknown property:', key, value);
```

Comment:
we should not display this message here, as this generic component does not pretend to handle all ac-props for a component, just the common ones